test/libsolidity/SolidityNameAndTypeResolution.cpp

```diff
@@ -5611,7 +5621,7 @@ BOOST_AUTO_TEST_CASE(using_interface_complex)
 	char const* text = R"(
 		interface I {
 			event A();
-			function f();
+			function f() returns(bool);
```

Comment:
Indeed, but I would venture to say it falls a bit out of the scope of this PR. This happens because `TypeChecker::checkContractIllegalOverrides` relies in `_contract.annotation().linearizedBaseContracts` which probably does not include the implemented interfaces.

Should I leave the test highlighting the broken functionality and submit another PR with the fix?